wbooks/src/app/screens/BookDetail/components/Information/styles.ts

```diff
@@ -37,5 +39,18 @@ export default StyleSheet.create<Styles>({
   },
   subTitle: {
     fontSize: SIZE.normal
+  },
+  successBtn: {
+    marginTop: 20,
+    alignSelf: 'center',
+    backgroundColor: COLORS.green,
+    alignItems: 'center',
+    borderRadius: 20,
+    padding: 5
+  },
+  successImage: {
+    height: 17,
+    width: 17,
```

Comment:
extract in a common constant since it's a square size element

Reply:
Done!